src/senaite/core/subscribers/contact.py

```diff
@@ -18,11 +18,15 @@
 # Copyright 2018-2024 by it's authors.
 # Some rights reserved, see README and LICENSE.
 
-from plone.app.users.browser.userdatapanel import UserDataConfiglet as Base
-from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 
-
-class UserDataConfiglet(Base):
-    """Control panel version of the userdata panel
+def on_contact_modified(contact, event):
+    """Event handler when a Contact was modified
     """
-    template = ViewPageTemplateFile("templates/account-configlet.pt")
+    user = contact.getUser()
```

Comment:
This `getUser` relies on Contact's `Username` field to get the user instead of relying on the `backreference` with key `linked_contact_uid`. Not for this PR probably, but to just keep in mind.

Reply:
Yes, that is correct.
However, this should be the more "safe" way instead of relying on the property I would say. Actually, I forgot about the `getUsername` index in the contact catalog...
I guess I will change the code to use this function instead of looking up the property to fetch the contact.

Reply:
Changed in 92d92e19